src/components/shared/SourceView-codemirror.js

```diff
@@ -243,8 +252,7 @@ export class SourceViewEditor {
         lineNumbers(),
         languageConf.of(_languageExtForPath(path)),
         syntaxHighlighting(classHighlighter),
-        EditorState.readOnly.of(true),
```

Comment:
I see that readOnly is now absent, is that expected?

From the snapshot below, I see `aria-readonly` is removed as a result, but this still seems relevant according to the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-readonly).

What do you think?

Reply:
I don’t know if aria-readonly provides any value on non-editable controls. All web content is readonly by default, this doesn’t need to be explicitly called out. I don’t consider the source view to be an editable control, it’s just like all the other content that the profiler displays, which is also read only.

Reply:
Oh, and yes, the removal was intentional, see my previous comment on the PR.

Reply:
Here is the doc for the [`readOnly` facet](https://codemirror.net/docs/ref/#state.EditorState%5EreadOnly):

> This facet controls the value of the [readOnly](https://codemirror.net/docs/ref/#state.EditorState.readOnly) getter, which is consulted by commands and extensions that implement editing functionality to determine whether they should apply. It defaults to false, but when its highest-precedence value is true, such functionality disables itself.
>
>    Not to be confused with [EditorView.editable](https://codemirror.net/docs/ref/#view.EditorView%5Eeditable), which controls whether the editor's DOM is set to be editable (and thus focusable).

I don't really mind about what's written there. My concern was especially about the property `aria-readonly` being removed, and therefore I was wondering whether this removal was necessary to fix this bug or if this was for something else.
But from a quick research I see that it's not exposed by screen readers currently, so well I won't argue too much about that :-)